ASP.NET minimal hosting model doc migration. (#10230)
auth0/docs: articles/quickstart/webapp/aspnet-core/interactive.md

### Configure the middleware

> To enable authentication in your ASP.NET Core application, use the middleware provided by the SDK. Under the `var builder = WebApplication.CreateBuilder(args);` method of your `Program` class call `builder.Services.AddAuth0WebAppAuthentication()` to register the SDK's middleware.

Reviewer: ```suggestion
To enable authentication in your ASP.NET Core application, use the middleware provided by the SDK. Go to the `Program.cs` file and call `builder.Services.AddAuth0WebAppAuthentication()` to register the SDK's middleware.
```

Ensure to configure the `Domain` and `ClientId`, these are required fields to ensure the SDK knows which Auth0 tenant and application it should use.

> Make sure you have enabled authentication and authorization below your `var app = builder.Build();` middleware pipeline setup statement in `Program.cs`.

Reviewer: ```suggestion
Make sure you have enabled authentication and authorization in your `Program.cs` file.
```
